src/main/resources/api.yaml

```diff
@@ -52,3 +52,18 @@ definitions:
     required:
       - username
       - admin
+  SprintState:
+    type: string
+    enum:
+      - upcoming
+      - declarable
+      - padding
+      - in_progress
+      - finished
+      - closed
+  UserRole:
```

Comment:
UserRole chyba lepiej tu przerzucić, skoro będzie zmieniany

Reply:
Ogarnia enumy. 

Nie wiem czy UserRole deleted ma sens - w takiej metodzie getUsers, która pobiera skład projektu, chyba wystarczy że będą istniejący użytkownicy?
Deleted przyda się przy przeglądaniu historii, żeby oznaczyć, że taki kiedyś był, ale już nie ma (bo jego dostępność musi zostać, a nie da się usunąć użytkownika bez zostawiania brzydkiego nulla w kluczu obcym).

Z drugiej strony, jeśli ponownie dodamy usuniętego użytkownika, to nie szukamy, czy jest dla niego UserInProject z deleted=1, tylko chyba tworzymy nowe?

Reply:
Hm, to tej pory mówiliśmy o `UserRole.DELETED`. `getUsers()` by musiało filtrować tylko aktywnych. Usuwanie `UserInProject` przy usuwaniu kogoś z projektu na pewno nie wchodzi w grę - można zrobić albo osobnego boola, albo takiego enuma.

> Z drugiej strony, jeśli ponownie dodamy usuniętego użytkownika, to nie szukamy, czy jest dla niego UserInProject z deleted=1, tylko chyba tworzymy nowe?
Do tej pory mówiliśmy właśnie o szukaniu  `UserInProject` z deleted=1